git-service/src/softserve/models.rs

```diff
@@ -21,3 +27,14 @@ pub struct CreateTokenResponse {
     pub token_name: String,
     pub token: String,
 }
+
+#[derive(Serialize)]
+pub struct CreateRepoResponse {
+    pub repo_name: String,
+    pub repo_url: String,
+}
+
+#[derive(Serialize)]
+pub struct SetupWebhookRequest {
+    pub repo_name: String,
+}
```

Comment:
```suggestion
#[derive(Deserialize)]
pub struct SetupWebhookRequest {
    pub repo_name: String,
}
```

This is intended to be used in a request so should derive `Deserialize`.

Reply:
Yea, I am not using it any more, will definitely clean it up